lib/cloud_controller/metrics/request_metrics.rb

```diff
@@ -5,17 +5,27 @@ module Metrics
     class RequestMetrics
       def initialize(statsd=Statsd.new)
         @statsd = statsd
+        @collect_metrics_for_routes = ['service_instances', 'service_bindings', 'service_keys']
       end
 
       def start_request
         @statsd.increment 'cc.requests.outstanding'
       end
 
-      def complete_request(status)
+      def complete_request(path, method, status)
         @statsd.batch do |batch|
           batch.decrement 'cc.requests.outstanding'
           batch.increment 'cc.requests.completed'
           batch.increment "cc.http_status.#{status.to_s[0]}XX"
+
+          if path.length > 4
```

Comment:
This feels a bit clumsy—but I'm not overly bothered. I feel like `path.start_with? "/v2/"` is more explicit and might do something less weird if that changes at some point in the future.

Reply:
Probably I'll change it to strip the first /xxxx part, as it can be /v2 or /v3

Reply:
Regex time? 💃

Reply:
fixed without regexes :)